python/cuml/benchmark/algorithms.py

```diff
@@ -246,6 +428,16 @@ def all_algorithms():
             accepts_labels=True,
             accuracy_function=cuml.metrics.accuracy_score,
         ),
+        AlgorithmFIL(
+            xgb if has_xgboost() else None, 
+            sklearn.ensemble.RandomForestClassifier,
+            tl if has_treelite() else None,
+            cuml.ForestInference,
+            shared_args={"max_depth": 10, "n_estimators": 10},
+            cuml_args=dict(algo="BATCH_TREE_REORG", output_class=True, threshold=0.50),
```

Comment:
Can I benchmark more FIL algorithms?

Reply:
Yes, this can be done by using `--cuml-param-sweep algo=[0,1,2]`, although the algo list cannot take strings as input (e.g. cannot use `algo=["NAIVE","TREE_REORG","BATCH_TREE_REORG"]`). I'll fix this tomorrow

Reply:
Yes, this can be done by using `--cuml-param-sweep algo=[0,1,2]`. However, for now, the parser doesn't support strings as input (so we cannot do `--cuml-param-sweep algo=["NAIVE"]`). I will add support for this tomorrow

Reply:
Using `--cuml-param-sweep 'algo=["NAIVE"]'` helps